Stripe/PublicHeaders/STPFile.h

```diff
@@ -48,7 +48,7 @@ typedef NS_ENUM(NSInteger, STPFilePurpose) {
 /**
  * Returns the string value for a purpose.
  */
-+ (NSString *)stringFromPurpose:(STPFilePurpose)purpose;
++ (nullable NSString *)stringFromPurpose:(STPFilePurpose)purpose;
```

Comment:
Ah I didn't realize about the unknown thing. So yeah I think this should be considered a nullabilty bugfix and not a breaking api change (although maybe still mention it in the migrating doc).